controllers/vpcendpoint/vpcendpoint_controller.go

```diff
@@ -51,9 +53,6 @@ type clusterInfo struct {
 	// clusterTag is the tag that uniquely identifies AWS resources for this cluster
 	// e.g. "kubernetes.io/cluster/${infraName}"
 	clusterTag string
-	// domainName is the domain name for the cluster's private hosted zone
-	// e.g. "${clusterName}.abcd.s1.devshift.org"
-	domainName string
```

Comment:
The domain name is no longer fixed to the cluster's Private Hosted Zone, so we will rely on `.status` instead for whatever the domain name ends up being